OsmAnd/res/values/strings.xml

```diff
@@ -1508,7 +1508,7 @@
     <string name="routing_attr_weight_description">Specify permitted vehicle weight limit on routes.</string>
     <string name="routing_attr_height_name">Height limit</string>
     <string name="routing_attr_height_description">Specify vehicle height to be permitted on routes.</string>
-    <string name="android_19_location_disabled">Since Android version 4.4 (KitKat) you cannot download and update maps to the old storage folder (%s). Copy all OsmAnd files to new storage location?
+    <string name="android_19_location_disabled">Android 4.4 (KitKat) and later versions can not download or update maps to the old storage folder (%s). Copy all OsmAnd files to new storage location?
```

Comment:
This changes the meaning of the message. It implies that Android itself cannot download files to the old storage folder, when the intent of the message is to explain that due to changes in Android, *OsmAnd* can't download files to the old storage folder. And again, "can not" should be "cannot".